main.lua

```diff
@@ -41,6 +48,7 @@ lib.load = function ()
   package.path = package.path .. ';love-api/?.lua'
   local API = require("love_api")
   printModule(lib, API, libTxt, "")
+  io.write("Status: "..missing.."/"..missing+have.." - "..string.format("%.0f", have/missing*100).."%", "\n")
```

Comment:
The computation of the percentage is not correct. It should be: 100 * have / (have + missing)